packages/tokamak/contracts-bedrock/test/L1StandardBridge.t.sol

```diff
@@ -14,11 +14,14 @@ import { Constants } from "src/libraries/Constants.sol";
 import { StandardBridge } from "src/universal/StandardBridge.sol";
 import { L2StandardBridge } from "src/L2/L2StandardBridge.sol";
 import { CrossDomainMessenger } from "src/universal/CrossDomainMessenger.sol";
+import { L1CrossDomainMessenger } from "src/L1/L1CrossDomainMessenger.sol";
 import { AddressAliasHelper } from "src/vendor/AddressAliasHelper.sol";
 
 // Target contract
 import { OptimismPortal } from "src/L1/OptimismPortal.sol";
 
+import "forge-std/console.sol";
```

Comment:
I think you used it for logging during developing unit test, so please clear it if you don't use it anymore.